content/en/docs/reference/kubectl/conventions.md

```diff
@@ -61,8 +61,10 @@ If you do not specify a generator flag, other flags prompt you to use a specific
 
 {{< note >}}
 These flags use a default generator only when you have not specified any flag.
-This means that when you combine `--generator` with other flags the generator that you specified later does not change. For example, in a cluster v1.4, if you initially specify
-`--restart=Always`, a Deployment is created; if you later specify `--restart=Always`
+This means that when you combine `--generator` with other flags the generator that you specified later does not change.
+
```

Comment:
I may have this out of context, but what happens: `When you combine the --generator option with other flags` ?
If you later use the --generator flag, this has no effect as the generator already was set by one of the other flags?